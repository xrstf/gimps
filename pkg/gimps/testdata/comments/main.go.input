package main

import (
	"log"
)

import (
	"fmt" // required!
)

func main() {
	fmt.Println("hello world")
	log.Println("foo")
}
