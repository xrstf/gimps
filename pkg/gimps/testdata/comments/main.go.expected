package main

import (
	"fmt" // required!
	"log"
)

func main() {
	fmt.Println("hello world")
	log.Println("foo")
}
